armi/cases/case.py

```diff
@@ -769,10 +770,13 @@ def copyInterfaceInputs(
         # guess. In future, it might be nice to have interfaces specify which
         # explicitly.
         for key, files in interfaceFileNames.items():
-            if isinstance(key, settings.Setting):
-                label = key.name
-            else:
-                label = key
+
+            if not isinstance(key, settings.Setting):
+                try:
+                    key = cs.getSetting(key)
+                except NonexistentSetting(key):
```

Comment:
Lets not do a pass here but include that failure that we had before with the Value error instead. This would just be a `raise ValueError("...blah")` instead of pass so that this does actually fail if the specify inputs is incorrect. Once you have this I'm good to merge in.

Reply:
@jakehader that makes sense! 

Would I need to also include a `runLog` error message or is the failing with `ValueError` enough?

Reply:
Failing is fine because that will emit the error message clearly. RunLog errors can be used in instances where say you are in a loop and want to emit many messages before failing. In this case because the failure is at the same place where you'd add a runLog too it's optional and not really needed in my opinion.